packages/gatsby/src/utils/tracer/index.ts

```diff
@@ -1,6 +1,6 @@
-const { slash } = require(`gatsby-core-utils`)
-const path = require(`path`)
-const opentracing = require(`opentracing`)
+import { slash } from "gatsby-core-utils"
+import path from "path"
+import opentracing from "opentracing"
```

Comment:
I think we need to import individual exports like this:
```suggestion
import { Tracer, initGlobalTracer } from "opentracing"
```
and adjust code in `initTracer` for those imports

Reply:
Thank you for giving me a hand !! 😊
I fix it.
https://github.com/gatsbyjs/gatsby/pull/22296/commits/0b953ff16e611634e49fe28bdaa889ff4aad8b19